polar_route/crossing_smoothing.py

```diff
@@ -1047,14 +1047,23 @@ def blocked(self,new_cell,cell_a,cell_b):
         end   = cell_b['SIC']
```

Comment:
Should the docstring here say 5% or 10%?

Reply:
Good point it should actually say something like 'depending on the blocking condition defined in the route config (defaults: 10)'. I will add this as a change with the regression tests coming soon